.env

```diff
@@ -49,3 +49,13 @@ HYKU_USER_DEFAULT_PASSWORD=password
 # Comment this line if the Groups with Roles feature is disabled and to
 # allow registered users to create curation concerns (Works, Collections, and FileSets)
 HYKU_RESTRICT_CREATE_AND_DESTROY_PERMISSIONS=true
+
+##START## Enable Google Analytics
+# Uncomment to enable and configure Google Analytics, see README for instructions.
+HYRAX_ANALYTICS=true
+# This is Kirk's test GA Measurement ID
+GOOGLE_ANALYTICS_ID=G-T2NVX8EX8B
```

Comment:
You probably won't want to commit this value; as everyone will be sending that information in Dev.  Frankly it should probably be "G-WEFAKINGIT" or some such non-sense (it needs to be `G-` then 10 chars)